addons/api/addon/models/alias.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import GeneratedAliasModel from '../generated/models/alias';
+
+export const TYPE_ALIAS = 'target';
```

Comment:
non-blocking: from my understanding,  think we are only going to support aliases for targets for now. Since in the future we may want to support more types for many resources we should also add `export const TYPES_ALIAS = Object.freeze([...]);` . And perhaps rename `TYPE_ALIAS` to `TYPE_ALIAS_TARGET`

Reply:
yeah that makes sense